### PR TITLE
Feature/news jobs views create

### PR DESCRIPTION
**Pull request by VMatiasDev**
What this PR includes:

* Adds images that will be used on news (until this moment)
* Adds proptypes on blogcards component
* Create news cards component
* Create jobs cards component
* Create topviews cards component
* Import everything to homepage (organization on the page is missing)

### CODE REVIEW

File: src/pages/Home/index.jsx [hunk 1]
```diff
@@ -1,40 +1,98 @@
...
+      text: '3 - O que é CSS?'
+    }
+  ];
+
   return (
     <div className={styles.wrapper}>
       <h1>Home SnakeTech</h1>
       <Challenge/>
+
       <div className={styles.cardsBlogContainer}>
```
Comment (viniciuscosmome) [inline src/pages/Home/index.jsx:71]: @VMatiasDev 
Verifica esse fundo branco do container de cards, ele deve cobrir a tela inteira horizontalmente.
<br />

image

**Review by viniciuscosmome — CHANGES_REQUESTED** [review]

**Review by MullerHub — DISMISSED** [review]
it's good

**Review by senegalha — DISMISSED** [review]
LGTM

**Review by viniciuscosmome — APPROVED** [review]

**Comment by M0nicaVaz** [comment]
LGTM

**Review by M0nicaVaz — APPROVED** [review]

**Pull request merged**